cli/process_poller_test.go

```diff
@@ -80,19 +81,19 @@ func TestProcessPollerPoll(t *testing.T) {
 
 	pp := ProcessPoller{BinPath: appPath}
 	pid, err := pp.Poll(ctx)
-	assert.NoError(t, err)
+	require.NoError(t, err)
 	assert.Equal(t, appPathPID, pid)
 
 	pp.BinPath = altPath
 	pid, err = pp.Poll(ctx)
-	assert.NoError(t, err)
+	require.NoError(t, err)
 	assert.Equal(t, altPathPID, pid)
 
 	pp.Interval = time.Millisecond
 	pp.BinPath = "/path/that/is/not/found"
 	ctx, cancel := context.WithTimeout(ctx, time.Second)
 	defer cancel()
 	pid, err = pp.Poll(ctx)
-	assert.ErrorIs(t, err, context.DeadlineExceeded)
+	require.ErrorIs(t, err, context.DeadlineExceeded)
```

Comment:
This seems undesired. We want a non-nil error, but also want the test to continue and report all other errors if this fails.

```suggestion
	assert.ErrorIs(t, err, context.DeadlineExceeded) //nolint:testifylint // Continue on failure.
```